network.go

```diff
@@ -143,6 +144,12 @@ func (network *_Network) _GetNodeAccountIDsForExecute() []AccountID { //nolint
 		nodes = append(nodes, network.healthyNodes[i].(*_Node).accountID)
 	}
 
+	// shuffle the nodes, so that the first node is not always the same
+	for i := range nodes {
+		j := rand.Intn(i + 1) // #nosec
+		nodes[i], nodes[j] = nodes[j], nodes[i]
+	}
+
```

Comment:
I think we should randomize the whole list before getting the 1/3rd of the nodes. So we can hit all of the nodes, not just the first third.